training pages

will-moore opened the pull request:
See https://trello.com/c/2VSqMQmj/18-website-training-page-update

Ready for first look at these pages.
Added link to training page from "About Us" menu and support page.

Will be staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/training/
once build is run.

cc @jburel @pwalczysko

jburel commented:
Link to events:
From the discussion we had. We said we will reformat ``presentation.md`` and point to that page
so they can info about length of workshop
Length is not captured.

Probably easier to sit and discuss

mtbc commented on training/index.html:
```diff
@@ -0,0 +1,75 @@
+                setup procedures used.
+            </p>
+            <p>
+                Our workshops range in length from a 1-hour overview of OMERO up to a full-day.
```
no hyphen for "full day"

will-moore commented:
Added a paragraph about workshop length.
We have links to the training-repos which now lists length of past workshops.

joshmoore commented on training/index.html:
```diff
@@ -0,0 +1,75 @@
+        </div>
+
+        <div class="row">
+            <div class="small-12 medium-3 columns">
```
This card looks larger on my screen.

will-moore replied:
That's because the text is longer. All those boxes expand to fit the text. We could try making the text a tiny bit shorter.

joshmoore commented on training/index.html:
```diff
@@ -0,0 +1,75 @@
+                    <div class="card-section">
+                        <i class="border-red icon-red fa fa-graduation-cap fa-2x"></i>
+                        <h4>Run a workshop</h4>
+                        <p class="card-caption">Find information on how to run your own workshop.</p>
```
I assume as soon as we have a second category of training, this text will explicitly mention "your own OMERO workshop"?

will-moore replied:
I guess we can decide that at the time. Either add another link from here or link to a /training/setup/ 
Maybe that would make a better name for the current /training/omero/ page if we're likely to add other non-omero training materials to the page later. @jburel ??

jburel replied:
we will other material e.g. training/idr
I think the name ``/training/omero`` is fine

joshmoore replied:
> I think the name /training/omero is fine

Agreed. I was more thinking there would be other pages as opposed to putting everything on one.

jburel commented on training/index.html:
```diff
@@ -0,0 +1,75 @@
+            <div class="row column text-center">
+                <h1>OME Training</h1>
+                <p>
+                    OME wants to promote understanding of our products with practical
```
I will rephrase it and replace ``OME wants`` by something like ``To promote understanding of our products, regular practical  workshops are organised.``

jburel commented on training/omero/index.html:
```diff
@@ -0,0 +1,72 @@
+                        <a href="https://github.com/ome/training-repos" target="_blank">Training
+                        Repos</a> is the parent repository that links to the others below and
+                        is tagged for each workshop we run.
+                        This allows us to know the precise version of all scripts and guides
```
and also software version (server/figure etc.) used

mtbc commented on training/index.html:
```diff
@@ -0,0 +1,75 @@
+                <h1>OME Training</h1>
+                <p>
+                    To promote understanding of our products, regular practical workshops
+                    are organised in the community.
```
organized

mtbc commented on training/omero/index.html:
```diff
@@ -0,0 +1,72 @@
+            <div class="row column text-center">
+                <h1>OMERO Workhop Setup</h1>
+                <p>
+                    All our training preparation materials are available to help you setup
```
"set up" (as verb)

mtbc commented on training/omero/index.html:
```diff
@@ -0,0 +1,72 @@
+        <hr>
+        <div class="row column">
+            <p>
+                The training resources are hosted on several github repositories:
```
GitHub

mtbc commented on training/omero/index.html:
```diff
@@ -0,0 +1,72 @@
+                    <li>
+                        <a href="https://github.com/ome/training-scripts" target="_blank">
+                        Training scripts</a> provides maintenance scripts for managing user accounts and
+                        data on the training server as well as scripts for user's workflows in the
```
users'

pwalczysko commented:
WE do not have any mention of the workshops in ELMI and TOPIM here ?

jburel commented:
No
The workshops are listed here https://github.com/ome/training-repos

will-moore commented:
Fixed @jrswedlow's comments at https://trello.com/c/2VSqMQmj/18-website-training-page-update#comment-5b8ad68d3b1ac4550b13fd68
Typo fixed in title, removed outreach server URL and added Support > Users/Training links.

jburel commented:
@will-moore I will add the events to https://github.com/openmicroscopy/www.openmicroscopy.org/pull/253
otherwise it will conflict

will-moore commented on _includes/navbar-main.html:
```diff
@@ -18,6 +18,7 @@
                             <li><a href="{{ site.baseurl }}/licensing/">Licensing</a></li>
                             <li><a href="{{ site.baseurl }}/citing-ome/">Citing OME</a></li>
                             <li><a href="{{ site.baseurl }}/artwork/">Artwork</a></li>
+                            <li><a href="{{ site.baseurl }}/training/">Training</a></li>
```
From @jrswedlow comment at https://trello.com/c/2VSqMQmj/18-website-training-page-update
"I see Training listed under About, and on the Support page. Seems it should exposed more. Can Support have a menu with two choices-- Users--> current Support page, Training goes to Training page? This is so important. Don't bury it."

will-moore commented on _includes/navbar-main.html:
```diff
@@ -40,7 +41,12 @@
-                    <li><a href="{{ site.baseurl }}/support/">Support</a></li>
+                    <li class="has-submenu"><a href="{{ site.baseurl }}/support/">Support</a>
+                        <ul class="submenu menu vertical" data-submenu>
+                            <li><a href="{{ site.baseurl }}/support/">Users</a></li>
```
I tried not having any link at the top but without the ```<a>``` the formatting is wrong and with ```<a href="#">``` the build fails since this is not a valid link.
It's also consistent with the other about/news/products links in the nav-bar in that you can click on the main link to go to the main about or news or products page. I guess we could remove the Support link from the submenu, but a sub menu with only a single item seems kinda wrong too.

sbesson replied:
I guess my primary concern is exactly the inconsistency with About and News links where to `/about` and `/news`  are only landing pages for all the content of the submenu. 

I do like the idea of putting `Training` alongside the support section. Would there be a way to split the content so that we have
1- a landing page
2- a `support/` page
3- a `training/` page

will-moore commented:
As discussed in slack, having menu top-links to pages can be confusing (and they don't work on mobile layout). Would be nice to move the landing page links to the first item in the submenus, e.g. 'Overview' in another PR.

jburel commented:
Merging and redeploying website
